README.md

```diff
@@ -1,136 +1,96 @@
 
-# **D**jango **R**epository for **ME**vo**L**ib **W**eb-based **G**raphical **U**ser **I**nterface
+# Django Repository for MEvoLib Web-based Graphical User Interface
 
 First of all, MEvoLib is a library of freely available Python tools for molecular evolution.
+
 This project has been developed based on the use of that library, keeping in mind to provide users:
-    - An easier and more visual way to interact with MEvoLib.
-    - A more controlled process, not only because of custom stages that will be run within the workflow (Fetch, Cluster, Align and Inference,
-    for now), but also by providing all the available parameters for each stage in a single sight, listing the possibilities for each one.
-    - A simplier way to get the results, as they will be stored in a ZIP that will be downloaded in the local machine after each requested
-    process has finished. Even if it fails, a filtered log message will be displayed, so that the user has a clue of what went wrong. 
+- An easier and more visual way to interact with MEvoLib.
+- A more controlled process, not only because of custom stages that will be run within the workflow (Fetch, Cluster, Align and Inference, for now), but also by providing all the available parameters for each stage in a single sight, listing the possibilities for each one.
+- A simpler way to get the results, as they will be stored in a ZIP that will be downloaded in the local machine after each requested process has finished. Even if it fails, a filtered log message will be displayed, so that the user has a clue of what went wrong.
 
-# **L**icense **T**erms
+## License terms
 
 This Django package is open source software made available under the Apache 2.0 License terms. Please see the LICENSE file for further details.
 
-# **I**nstallation **P**rocess
+## Installation
 
-- The first step is to clone this library. You can do it by typing this line in your console:
+The first step is to clone this library. You can do it by typing this line in your console:
 ```bash
 git clone https://github.com/nachorequeno/mevolib-ui.git
 ```
 
-- After that, you have to enter to the folder:
+Next, you have to install all of the dependencies:
 ```bash
-cd mevolib-ui
+pip install mevolib-ui/MEvoLibGUI/.
```

Comment:
After some research, I am not sure whether it is the correct way to do it or not, as with the "-e" flag, you do it in editable mode. This means every change the packages suffer will be reflected on your work. I reckon it is a good practice to use it.
Please, take a look at this discussion: 
https://stackoverflow.com/questions/35064426/when-would-the-e-editable-option-be-useful-with-pip-install

Reply:
Added `-e` back 👍